cmd/generate/codeowners/output.go

```diff
@@ -8,18 +8,30 @@ import (
 	"strings"
 
 	"github.com/open-sauced/pizza-cli/pkg/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 )
 
-func generateOutputFile(fileStats FileStats, outputPath string, ownersStyleFile bool, config *config.Spec) error {
+func generateOutputFile(fileStats FileStats, outputPath string, opts *Options, cmd *cobra.Command) error {
 	// Open the file for writing
 	file, err := os.Create(outputPath)
 	if err != nil {
 		return fmt.Errorf("error creating %s file: %w", outputPath, err)
 	}
 	defer file.Close()
+  var flags []string
+
+  cmd.Flags().Visit(func(f *pflag.Flag) {
+    flags = append(flags, fmt.Sprintf("--%s %v", f.Name, f.Value.String()))
+  })
+  generatedCommand := fmt.Sprintf("# $ pizza generate codeowners %s ", opts.path)
```

Comment:
`opts.Path` will be a full, system agnostic, absolute path. It shows up like:

```
# Generated with command:
# $ pizza generate codeowners /Users/jpmcb/workspace/opensauced/pizza-cli
```

It'd be ideal if we could just take the last part of that and add a slash at the end:

```
# Generated with command:
# $ pizza generate codeowners pizza-cli/
```

You should be able to use `filepath.Base()`:

```suggestion
  generatedCommand := fmt.Sprintf("# $ pizza generate codeowners %s/ ", filepath.Base(opts.path))
```